app/lnd/config/index.js

```diff
@@ -35,15 +52,15 @@ switch (plat) {
 }
 
 if (process.env.NODE_ENV === 'development') {
-  lndPath = join(__dirname, '..', '..', '..', 'resources', 'bin', plat, lndBin)
+  lndPath = join(appRootPath(), 'resources', 'bin', plat, lndBin)
```

Comment:
wanna dry this up by declaring `appRootPath` a const above? then we don't have to call the func multiple times.